ckks/bootstrapping/dummy_bootstrapping.go

```diff
@@ -0,0 +1,49 @@
+package bootstrapping
+
+import (
+	"fmt"
+
+	"github.com/tuneinsight/lattigo/v4/ckks"
+	"github.com/tuneinsight/lattigo/v4/rlwe"
+)
+
+// DummyBootstrapper is a dummy bootstrapper that complies to
+// the rlwe.Bootstrapping interface
+type DummyBootstrapper struct {
```

Comment:
DummyBootstrapper is an implementation of the rlwe.Bootstrapping interface that uses the secret-key to decrypt and re-encrypt the bootstrapped ciphertext.

Reply:
done